js/pricing.js

```diff
@@ -9,17 +12,44 @@ $(
     function () {
         const $confirmPersonalInformation = $('#confirm-personal-information');
         const $confirmDevelopmentAgreement = $('#confirm-development-agreement');
-        const $disabledButton = $('#order-now-btn');
+        const $orderButton = $('#order-now-btn');
+        const $redirect = $('#redirect');
+        const $loader = $('.redirect-loader', '#redirect');
+        const $errorContainer = $('.redirect-error', '#redirect');
+        const $tryAgain = $('#try-again');
+
+        const orderUrl = "{{site.data.backend_links.orderUrl}}";
+        const prodApiUrl = "{{site.data.backend_links.prodApiUrl}}";
+        const devApiUrl = "{{site.data.backend_links.devApiUrl}}";
+        const registerTransactionPath = "{{site.data.backend_links.registerTransactionPath}}";
+        // `window.mode` we setup in the `getting-help/service-section.html` file by Jekyll.
+        const apiUrl = window.mode === "development" ? devApiUrl : prodApiUrl;
+
 
         $confirmPersonalInformation.change(function () {
-            confirmAgreement(this, $disabledButton);
+            confirmAgreement($orderButton);
         });
 
         $confirmDevelopmentAgreement.change(function () {
-            confirmAgreement(this, $disabledButton);
+            confirmAgreement($orderButton);
+        });
+
+        $orderButton.click(function() {
+            submitOrder();
+        });
+
+        $tryAgain.click(function(event) {
+            event.preventDefault();
+            hideRedirect ();
+            submitOrder();
         });
```

Comment:
Please do not forget to format the code. I'd recommend checking the autoformat checkbox in WebStorm. Also, consider using arrow functions for cases where the function in a simple event handler, and there's no value in having a dedicated named function.

```suggestion
        $tryAgain.click(e => {
            e.preventDefault();
            hideRedirect();
            submitOrder();
        });
```